PluginBuilder/Views/Shared/_PublicHeader.cshtml

```diff
@@ -1,13 +1,29 @@
+@{
+    var isAuth = User.Identity?.IsAuthenticated == true;
```

Comment:
<img width="848" height="242" alt="image" src="https://github.com/user-attachments/assets/dcba18e3-2363-4e78-a1c2-fd0ed92102ad" />

errmm.. I think the positioning is cause of the "required" {not checked code tho)


Also if you can outrightly prevent a plugin owner from having to see the input field or the submit button to review

Reply:
<img width="768" height="395" alt="image" src="https://github.com/user-attachments/assets/d4eca528-9e1c-4218-b4e0-f0dabd7fb439" />


This dropdown looksn like it is being cut. tried with multiple version?

also there is shadow on the left, and not on the rtight